src/EmptyState/EmptyState.types.ts

```diff
@@ -0,0 +1,23 @@
+export type EmptyStateProps = {
```

Comment:
```suggestion
/**
 * This type defines the props for the EmptyState component.
 */
export type EmptyStateProps = {
```
missing comment descripting the type